README.md

```diff
@@ -1,36 +1,16 @@
 # Roadmap
-## Process and Tools
-The Notary v2 team uses the [Roadmap GitHub Project Board](https://github.com/notaryproject/roadmap/projects/1) to track short term (1-3 months) and medium term (3-6 months) efforts and long term (6-12 months). Once a broad objective (from the "Future/Backlog" column) is further scoped into a GitHub issue that has a clear objective, background and motive, it is moved into the "To do" column. From there, the Notary v2 team plans its releases via [GitHub Milestones](https://github.com/notaryproject/roadmap/milestones).
 
-## Current and Future Objectives
-At this time, the Notary community is working on Notary v2 specification  and two separate sub projects, TUF and Notation. This roadmap is focused  on the Notary v2 specification and its reference implementation in Notation client
+This repository is **not** in active maintenance. The original purpose is using this repository for tracking short, medium and long term enhancements for Notary Project. Now the Notary Project community uses the [project board](https://github.com/orgs/notaryproject/projects/10) for project planning and status tracking. You can also use GitHub milestones to track the releases of each repository:
 
-### Current (Short term) Focus
-- Collaborate with other open source projects for co-related/dependent features needed for signature verification with other open source clients
+- [notary TUF-based implementation milestones](https://github.com/notaryproject/notary/milestones)
```

Comment:
Is this up-to-date?

Reply:
I removed this line since `notary` TUF base implementation is not active maintained, so no need to show users the milestones.